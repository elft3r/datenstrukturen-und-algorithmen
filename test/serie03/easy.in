10
5 3 1 3 1 4
5 -5 -7 -7 2 -10
10 4 2 2 9 9 4 5 2 9 7
10 2 5 2 -9 7 -6 3 -3 8 -1
15 3 10 5 4 11 14 12 1 5 12 2 12 2 10 1
15 12 8 1 -2 0 12 1 4 10 14 7 -3 2 -9 -2
20 2 11 16 11 9 16 10 18 14 15 11 17 6 7 6 14 10 2 11 7
20 -3 2 -6 15 17 11 9 19 6 9 -3 -6 1 3 -8 10 -5 15 1 1
25 10 17 19 23 18 2 18 24 21 18 4 15 12 12 23 7 14 14 12 24 17 14 6 7 17
25 21 21 19 4 10 -8 28 13 11 9 14 13 19 -4 10 -7 10 -1 16 7 6 20 2 2 -5
