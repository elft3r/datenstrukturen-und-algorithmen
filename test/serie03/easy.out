3 3 3 3 4
1 1 3 3 4
-5 -5 -7 2 -5
-10 -7 -7 -5 2
4 4 2 9 9 4 5 9 9 9
2 2 2 4 4 5 7 9 9 9
2 5 2 5 7 2 3 5 8 7
-9 -6 -3 -1 2 2 3 5 7 8
3 10 5 10 11 14 12 10 5 12 11 14 12 12 10
1 1 2 2 3 4 5 5 10 10 11 12 12 12 14
12 12 8 12 1 12 8 12 10 14 7 12 2 8 -2
-9 -3 -2 -2 0 1 1 2 4 7 8 10 12 12 14
2 11 16 11 11 16 16 18 14 15 11 17 16 16 7 18 14 14 11 15
2 2 6 6 7 7 9 10 10 11 11 11 11 14 14 15 16 16 17 18
-3 2 -3 15 17 11 9 19 15 17 9 11 1 9 3 19 10 15 15 17
-8 -6 -6 -5 -3 -3 1 1 1 2 3 6 9 9 10 11 15 15 17 19
10 17 19 23 18 19 18 24 23 18 18 19 15 18 23 24 21 23 14 24 18 18 17 19 17
2 4 6 7 7 10 12 12 12 14 14 14 15 17 17 17 18 18 18 19 21 23 23 24 24
21 21 21 21 19 21 28 21 13 19 14 21 19 28 10 21 11 13 16 19 10 20 14 21 13
-8 -7 -5 -4 -1 2 2 4 6 7 9 10 10 10 11 13 13 14 16 19 19 20 21 21 28
